docs/_sources/api.rst.txt

```diff
@@ -25,6 +25,7 @@ API Reference
 
 :mod:`pysal.model`: Spatial Statistical Models
 ==============================================
+* `access: Spatial Accessibility  <https://access.readthedocs.io/en/latest/>`_
```

Comment:
This should be https://access.readthedocs.io/en/latest/api.html?

Reply:
Same with line 28 in `docsrc/api.rst`. I'll go ahead and make those changes here.